Sources/SpeziOnboarding/ConsentView/ConsentDocument.swift

```diff
@@ -39,6 +39,9 @@ import SwiftUI
 /// )
 /// ```
 public struct ConsentDocument: View {
+    /// The maximum width such that the drawing canvas fits onto the PDF.
+    static let maxWidthDrawing: CGFloat = 550
```

Comment:
Is there any scenario in which it might make sense to configure this value?

Reply:
Not really. This is currently a hard limit for us to be sure that the signature canvas fits onto the PDF. And you can always use a `.frame` modifier outside the view to make it smaller. This this is basically the maximum we currently support. Which is a hard limit as of right now due to our internal export logic.

Reply:
Thanks for the clarification! I think it would make sense to also include this in the inline DocC docs 🚀